.vs/ProjectSettings.json

```diff
@@ -0,0 +1,3 @@
+{
```

Comment:
Unless you have a good reason the `.vs` folder shouldn't be checked into source control.